### PR TITLE
Issue#60 - Anime data analysis and prediction

### PR DESCRIPTION
**Pull request by anoushka1405**
Issue #60 
Implemented a Random Forest regression model to predict the Rating based on numeric features in the dataset.

Used Rating as the target variable and all other numeric columns (except Rating) as features.
The Random Forest model was trained on the training set and predictions were made on the test set.

Changes:
Split the dataset into training and testing sets using train_test_split.
Initialized and trained a RandomForestRegressor model.
Made predictions for the test set using the trained model.

Screenshot 2025-01-29 at 7 13 39 PM

@PJ40200 @ISaumya1011 Please review

### CODE REVIEW

**Comment by anoushka1405** [comment]
@PJ40200 @ISaumya1011 Please review
